src/services/rfqm_service.ts

```diff
@@ -471,6 +491,70 @@ export class RfqmService {
         return this._blockchainUtils.isWorkerReadyAsync(workerAddress, gasPrice);
     }
 
+    public async getOrderStatusAsync(orderHash: string): Promise<StatusResponse | null> {
+        const transformSubmission = (submission: RfqmTransactionSubmissionEntity) => {
+            const { transactionHash: hash, createdAt } = submission;
+            if (!hash) {
```

Comment:
`return hash ? { hash, timestamp: createdAt.getTime()} : null;`

^ optional suggestion

Reply:
LOVE the swagger